### PR TITLE
i18n - internationalization

### PR DESCRIPTION
**Pull request by honza889**
Internationalization - exceptions with message moved into ElytronMessages
* Grouped by packages, ids of messages prefixed by group

### CODE REVIEW

File: src/main/java/org/wildfly/security/_private/ElytronMessages.java [hunk 1]
```diff
@@ -71,677 +75,982 @@
...
-    CertificateException notTrusted(Principal principal);
+    @Message(id = 6, value = "Unrecognised algorithm \"%s\"")
+    IllegalArgumentException unrecognisedAlgorithm(String algorithm);
 
-    @Message(id = 7, value = "No module found for identifier \"%s\"")
+    @Message(id = 7, value = "Cannot instantiate self-referential factory")
+    IllegalStateException cannotInstantiateSelfReferentialFactory();
+
+    @Message(id = 8, value = "Unexpected trailing garbage in X.500 principal")
+    IllegalArgumentException unexpectedTrailingGarberageInX500principal();
```
Comment (dmlloyd) [inline src/main/java/org/wildfly/security/_private/ElytronMessages.java]: Garberage -> Garbage

Comment (dmlloyd) [inline src/main/java/org/wildfly/security/_private/ElytronMessages.java]: If you're using IntelliJ IDEA, enable spell-checking and recheck this file.

**Comment by honza889** [comment]
Spellchecked.

**Comment by dmlloyd** [comment]
Overall the gist of this looks good.  I won't analyze every single code change on this one though.

**Comment by darranl** [comment]
Also not reviewed 100% line by line - I am going to merge but there were a couple of cases where we could probably use Assert from common.

**Pull request merged**